core/unix/inc/TUnixSystem.h

```diff
@@ -71,6 +78,17 @@ class TUnixSystem : public TSystem {
    static int          UnixRecv(int sock, void *buf, int len, int flag);
    static int          UnixSend(int sock, const void *buf, int len, int flag);
 
+   // added helper static members for stacktrace
+   static const int                    fPidStringLength = 255;
+   static char                         fPidString[fPidStringLength];
+   static int                          fParentToChild[2];
+   static int                          fChildToParent[2];
+   static std::unique_ptr<std::thread> fHelperThread;   
+   static char *const                  kStackArgv[];
+   static char *const *                GetStackArgv();
```

Comment:
This is a set of tightly related static, consider wrapping them into a private struct.